x-pack/packages/observability/alert_details/src/components/alert_threshold_annotation.tsx

```diff
@@ -6,31 +6,39 @@
  */
 
 import React from 'react';
+import { i18n } from '@kbn/i18n';
 import { AnnotationDomainType, LineAnnotation } from '@elastic/charts';
-import { CHART_ANNOTATION_RED_COLOR } from '../constants';
 
-export function AlertThresholdAnnotation({
-  threshold,
-}: {
-  threshold?: number;
-}) {
-  if (!threshold) return <></>;
+interface Props {
+  id: string;
+  threshold: number;
+  color: string;
+}
+
+const ANNOTATION_TITLE = i18n.translate(
+  'observabilityAlertDetails.alertThresholdAnnotation.detailsTooltip',
+  {
+    defaultMessage: 'Alert started',
+  }
+);
 
+export function AlertThresholdAnnotation({ threshold, color, id }: Props) {
```

Comment:
@fkanout Is this only for the APM Latency rule? If that is the case, we should not move it to the `alert_details_package` package as it will be included in the bundle for other alert details pages as well.

Same for `AlertThresholdTimeRangeRect`

Reply:
@maryam-saeidi, I think It's not only about the APM. As we do the rule combination, it will be needed for all other rule types.

Reply:
@maryam-saeidi, for the combined rules, we will use another/our chart that does not ship annotations out of the box like the one from Infra (will use Lens, likely). Also, even with the combination we still have other alert details pages.
 
I see `@kbn/observability-alert-details` package as a toolbox for the alert details pages, we use what it provides once it's needed. 

I think there is no harm in adding these components now, especially, since we have ownership over them if we want to update anything. Later, If there is no use for them, we can delete them anytime,

Reply:
OK, let's do that 👍🏻